application/templates/deployment.yaml

```
@@ -246,13 +246,10 @@ spec:
         {{- end }}
         {{- end }}
       {{- end }}
+        {{- if .Values.deployment.resources }}
```

We can turn this into a `with` block instead! (`.`) would be used within.

```suggestion
        {{- with .Values.deployment.resources }}
```